tests/integration/commands/test_describe.py

```diff
@@ -48,7 +48,7 @@ def test_describe_topic_last_timestamp_does_not_commit(
     # cannot use pytest.raises(ConsumerGroupDoesNotExistException) because other tests may have committed offsets
     # for this group
     try:
-        data = consumergroup_controller.get_consumer_group(config.ESQUE_GROUP_ID).describe(verbose=True)
+        data = consumergroup_controller.get_consumer_group(config.ESQUE_GROUP_ID).describe(partitions=True)
```

Comment:
Could you add an assertion for the new field somewhere? Or maybe produce something to two partitions and verify that the timestamps match (or that they are at least different)?

Reply:
Done